### PR TITLE
EB corbos Linux Release 1.1 Q1/24

### PR DESCRIPTION
**Pull request by thir820**

### CODE REVIEW

**Review by ulbu261833 — DISMISSED** [review]
sysroot build is failing for x86_64 when running the commands
box_build_sysroot.sh /build/workspace/images/qemu-systemd-x86_64/appliance.kiwi 
or
box_build_sysroot.sh /build/workspace/images/qemu-crinit-x86_64/appliance.kiwi
See related build log files
[qemu_crinit_x86_64_result.log](https://github.com/Elektrobit/ebcl_template/files/14861515/qemu_crinit_x86_64_result.log)
[qemu_systemd_x86_64_result.log](https://github.com/Elektrobit/ebcl_template/files/14861516/qemu_systemd_x86_64_result.log)

**Comment by thir820** [comment]
> sysroot build is failing for x86_64 when running the commands box_build_sysroot.sh /build/workspace/images/qemu-systemd-x86_64/appliance.kiwi or box_build_sysroot.sh /build/workspace/images/qemu-crinit-x86_64/appliance.kiwi See related build log files [qemu_crinit_x86_64_result.log](https://github.com/Elektrobit/ebcl_template/files/14861515/qemu_crinit_x86_64_result.log) [qemu_systemd_x86_64_result.log](https://github.com/Elektrobit/ebcl_template/files/14861516/qemu_systemd_x86_64_result.log)

Should be fixed now. Seems the package dependencies don't pull libc6-dev into prod or dev for x96_64.

File: images/berrymill.template [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      
+      EBcL_1.1_dev:
+        # Security maintained packages for usage during development phase
+        url: http://linux.elektrobit.com/eb-corbos-linux
+        type: apt-deb
+        key: file:///etc/berrymill/keyrings.d/repo_eb_gm.pub.gpg
+        name: ebcl_1.1
+        components: dev
+
+      EBcL_1.1_dev:
```
Comment (ulbu261833) [inline images/berrymill.template:30]: shouldn't this be  EBcL_1.1_all?

File: images/berrymill.conf [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      
+      EBcL_1.1_dev:
+        # Security maintained packages for usage during development phase
+        url: http://linux.elektrobit.com/eb-corbos-linux
+        type: apt-deb
+        key: file:///etc/berrymill/keyrings.d/repo_eb_gm.pub.gpg
+        name: ebcl_1.1
+        components: dev
+
+      EBcL_1.1_dev:
```
Comment (ulbu261833) [inline images/berrymill.conf:30]: shouldn't this be EBcL_1.1_all?

Comment (thir820) [reply images/berrymill.conf]: Yes, fixed.

**Review by ulbu261833 — DISMISSED** [review]

**Comment by ulbu261833** [comment]
Shouldn't the change of the local config move be done also at the nautilos repo?

**Comment by ulbu261833** [comment]
Is this PR obsolete with PR https://github.com/Elektrobit/ebcl_template/pull/11?

**Comment by thir820** [comment]
> Is this PR obsolete with PR #11?

Yes.

**Comment by thir820** [comment]
Replaced by 1.1.1 which contains some fixed.

**Pull request closed without merging**